api.yml

```diff
@@ -264,6 +265,28 @@ paths:
         '401':
           $ref: "#/components/responses/Unauthorized"
 
+  /user/{idUser}/reminder/discard/{idReminder}:
```

Comment:
/user/{idUser}/reminder/{idReminder}/discard makes more sense but not a problem

Reply:
Just fixed this!